src/meta/test/meta_app_envs_test.cpp

```diff
@@ -140,7 +140,18 @@ TEST_F(meta_app_envs_test, update_app_envs_test)
          "",
          "200"},
         {replica_envs::BUSINESS_INFO, "300", ERR_OK, "", "300"},
-        {replica_envs::DENY_CLIENT_WRITE, "400", ERR_OK, "", "400"},
+        {replica_envs::DENY_CLIENT,
+         "400",
+         ERR_INVALID_PARAMETERS,
+         "Invalid deny client args, valid include: reject*all, reject*write, reject*read; "
+         "delay*all, delay*write, delay*read",
+         "400"},
```

Comment:
There are some paths in `check_deny_client` not covered, better to add more test cases.